task/fossa-scanner/0.1/fossa-scanner.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: fossa-scanner
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Security
+    tekton.dev/tags: CLI, fossa
+    tekton.dev/displayName: "fossa scanner"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    fossa-cli is a zero-configuration polyglot dependency analysis tool. You can point fossa CLI at any codebase or build,
+    and it will automatically detect dependencies being used by your project.
+
+    This task can be used to scan for dependency analysis, license scanning, vulnerability scanning,
+    attribution report generation, and more on the source code in standalone mode.
+  workspaces:
+    - name: source
+      description: A workspace that contains the source code to scan.
+  params:
+    - name: FOSSA_API_KEY
+      description: API key to be passed to fossa cli.
+      type: string
```

Comment:
I guess we should be taking this via secrets so that it doesn't gets compromised. WDYT?

Reply:
usually apikey is shard in an org to do code scan, but it's reasonable to put in secret, let me do some change